Assets/MixedRealityToolkit.SDK/Experimental/Features/UX/BoundsControl/Visuals/BoundsControlLinks.cs

```diff
@@ -0,0 +1,264 @@
+
+        /// <summary>
+        /// Material used for wireframe display
+        /// </summary>
+        public Material WireframeMaterial
+        {
+            get { return wireframeMaterial; }
+            set
+            {
+                if (wireframeMaterial != value)
+                {
+                    wireframeMaterial = value;
+                    SetMaterials();
+                    configurationChanged.Invoke();
+                }
+            }
+        }
+
+        [SerializeField]
+        [Tooltip("Radius for wireframe edges")]
+        private float wireframeEdgeRadius = 0.001f;
+
+        /// <summary>
+        /// Radius for wireframe edges
+        /// </summary>
+        public float WireframeEdgeRadius
+        {
+            get { return wireframeEdgeRadius; }
+            set
+            {
+                if (wireframeEdgeRadius != value)
+                {
+                    wireframeEdgeRadius = value;
+                    configurationChanged.Invoke();
+                }
+            }
+        }
+
+        [SerializeField]
+        [Tooltip("Shape used for wireframe display")]
+        private WireframeType wireframeShape = WireframeType.Cubic;
+
+        /// <summary>
+        /// Shape used for wireframe display
+        /// </summary>
+        public WireframeType WireframeShape
+        {
+            get { return wireframeShape; }
+            set
+            {
+                if (wireframeShape != value)
+                {
+                    wireframeShape = value;
+                    configurationChanged.Invoke();
+                }
+            }
+        }
+
+        [SerializeField]
+        [Tooltip("Show a wireframe around the bounds control when checked. Wireframe parameters below have no effect unless this is checked")]
+        private bool showWireframe = true;
+
+        /// <summary>
+        /// Show a wireframe around the bounds control when checked. Wireframe parameters below have no effect unless this is checked
+        /// </summary>
+        public bool ShowWireFrame
+        {
+            get { return showWireframe; }
+            set
+            {
+                if (showWireframe != value)
+                {
+                    showWireframe = value;
+                    configurationChanged.Invoke();
+                }
+            }
+        }
+
+        #endregion Serialized Properties
+
+        internal protected UnityEvent configurationChanged = new UnityEvent();
+
+        private List<Transform> links = new List<Transform>();
+        private List<Renderer> linkRenderers = new List<Renderer>();
+
+
+        internal void Clear()
+        {
+            if (links != null)
+            {
+                foreach (Transform transform in links)
+                {
+                    GameObject.Destroy(transform.gameObject);
+                }
+                links.Clear();
+            }
+        }
+
+        private void SetMaterials()
```

Comment:
This seems similar to BoundsControlHandlesBase.SetMaterials, could we combine this somehow?

Reply:
didn't realize all of them are actually using the same values - moved into util class